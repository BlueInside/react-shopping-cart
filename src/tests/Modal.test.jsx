import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';
import Modal from '../components/Modal';

describe('Modal component', () => {
  it('renders Modal correctly', () => {
    const { container } = render(<Modal />);

    expect(container).toMatchSnapshot();
  });
});
