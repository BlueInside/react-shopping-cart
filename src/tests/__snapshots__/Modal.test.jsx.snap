// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Modal component > renders Modal correctly 1`] = `
<div>
  <div />
</div>
`;
